sample-apps/endoscopy/lib/configs/inbody.py

```diff
@@ -35,12 +35,12 @@ def init(self, name: str, model_dir: str, conf: Dict[str, str], planner: Any, **
         super().init(name, model_dir, conf, planner, **kwargs)
 
         bundle_name = "endoscopic_inbody_classification"
-
+        version = conf.get("inbody")
```

Comment:
Hi @tangy5 , could we transfer from using hard coded strings ("inbody", "endoscopic_inbody_classification") into arguments? Although this folder belongs to `sample-apps`, it would be easier to maintain it if having less hard coded varibles.

Reply:
yes and no. Here the string is closely associate with the python script. The model can only load the "endoscopic_inbody_classification" bundle for example. It can't be used to load another bundle. This is the case for endoscopy and pathology only. 

But we could make it a default. If users want to use a different bundle name, it can be possible, but we are not recommend to change the name.